src/language-js/print/flow.js

```diff
@@ -252,6 +252,15 @@ function printFlow(path, options, print) {
         ")",
       ];
 
+    case "TypePredicate":
+      return [
+        node.asserts ? "asserts " : "",
+        print("parameterName"),
+        node.typeAnnotation
+          ? [" is ", printTypeAnnotationProperty(path, print)]
+          : "",
+      ];
```

Comment:
Please extract code for `TSTypePredicate` print, and reuse it instead of copy it. For now we put them in `type-annotation.js`